src/python_testing/TC_IDM_2_2.py

```diff
@@ -0,0 +1,155 @@
+#
```

Comment:
General comment - it would also be good to add in negative testing here - confirm both that the expected data is present, but also that the expected returned data is the ONLY data.

Reply:
It doesn't appear that negative test cases are included in the [test plan](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/interactiondatamodel.adoc), let alone in IDM 2-2.

Reply:
We can revisit this as needed; resolved for now to allow merge